### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by SlaviaF**

### CODE REVIEW

File: javascript/javascript2/week2/index2.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+</head>
+
+<body>
+
+
+
+
+
+
+
```
Comment (cameronjdavis) [inline javascript/javascript2/week2/index2.html:18]: So many blank lines that can be deleted. https://ibb.co/MPXjDVD
```suggestion
```

File: javascript/javascript2/week2/main.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
+body {
+    font-family: "Poppins", sans-serif;
+    /*background-color: #F9FBFD;*/
+    background-image: 
+    linear-gradient(to right, rgba(38, 49, 8, 0.52), rgba(240, 189, 20, 0.73)),
+    url("images/clock.jpg");
+background-position:0vh;
+color:white;
```
Comment (cameronjdavis) [inline javascript/javascript2/week2/main.css]: Consistent indentation.
```suggestion
    background-position:0vh;
    color:white;
```

File: javascript/javascript2/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+});
+
+
+
+searchInput.addEventListener('input', searchProducts);
+function searchProducts() {
+
+    productDisplayUl = document.getElementsByClassName('productDisplayList');
+    value = searchInput.value.toUpperCase();
+    let getProducts = products.filter(product => product.name.toUpperCase().includes(value));
```
Comment (cameronjdavis) [inline javascript/javascript2/week2/main.js]: When I search for a product by string, e.g. "computer", then type in a price, my filter string is ignored. Notice on this line you are filtering the full product list so any other active filters are ignored when a new filter is used. I'd like to be able to search for "computer" that cost less than 1000.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week2/main.js (hunk 3 above)
Comment (SlaviaF) [reply javascript/javascript2/week2/main.js]: Thank you CameonI will try to implement it

File: javascript/javascript2/week2/index2.html (hunk 1 above)
Comment (SlaviaF) [reply javascript/javascript2/week2/index2.html:18]: hahaha.....the cute dog seems unhappy. I will be careful the next time

File: javascript/javascript2/week2/main.css (hunk 2 above)
Comment (SlaviaF) [reply javascript/javascript2/week2/main.css]: will be careful the next time

**Pull request merged**